orchagent/orch.cpp

```diff
@@ -685,6 +685,49 @@ Executor *Orch::getExecutor(string executorName)
     return NULL;
 }
 
+bool Orch::handleSaiCreateStatus(sai_api_t api, sai_status_t status)
+{
+    /*
+     * This function aims to provide coarse handling of failures in sairedis create
+     * operation (i.e., notify users by throwing excepions when failures happen).
+     * TODO: 1. Add general handling logic for specific statuses (e.g., SAI_STATUS_ITEM_ALREADY_EXISTS)
+     *       2. Develop fine-grain failure handling mechanisms specific and replace
+     *          this coarse handling in each orch.
+     *       3. Take the type of sai api into consideration.
+     */
+    SWSS_LOG_THROW("Encountered failure in create operation, status: %d", status);
```

Comment:
>status [](start = 74, length = 6)

You may also log the api info.

Reply:
make %s and use sai_serialize_status(status).c_str() for string status

Reply:
Added api info in the log and updated to use sai_serialize functions for api and status. It appears that cfgmgr also includes orch as a source, it is needed to include saimetadata library for cfgmgr.